packages/inngest/src/components/connect/index.ts

```diff
@@ -261,7 +258,7 @@ class WebSocketWorkerConnection implements WorkerConnection {
     };
 
     const functions: Array<FunctionConfig> = this.options.functions.flatMap(
-      (f) => f["getConfig"](new URL("http://example.com")) // refactor; base URL shouldn't be optional here; we likely need to fetch a different kind of config
+      (f) => f["getConfig"](new URL("wss://connect"), undefined, true)
```

Comment:
The `appPrefix` here should be the ID of the client:
```suggestion
      (f) => f["getConfig"](new URL("wss://connect"), this.inngest.id, true)
```
Actually we should just quicky refactor this to be a single opts arg - this is tricky to read atm.